### PR TITLE
Ensures sentienced animals know how to speak all the languages their master does

### PR DESCRIPTION
**Pull request by PKPenguin321**
Fixes https://github.com/tgstation/tgstation/issues/32499

### CODE REVIEW

**Review by imtakingabreakdontatme — DISMISSED** [review]
It should grant the languages of the person using the sentience potion

**Comment by KorPhaeron** [comment]
Probably. I don't know how you'd choose the correct one to grant otherwise.

**Comment by PKPenguin321** [comment]
Seems exploitable with the librarian to me, are you sure?

**Comment by KorPhaeron** [comment]
I guess that is a fair concern but it'd also be weird if we ever had something that couldn't speak common sentience potion a mob

**Comment by PKPenguin321** [comment]
I guess if you insist
I'll touch it up either tomorrow or the day after

**Comment by PKPenguin321** [comment]
Still seems weird for edge cases to me, but I'm just going to do what Kor said.

**Comment by PKPenguin321** [comment]
That oughtta do it

File: code/modules/research/xenobiology/xenobiology.dm [hunk 1]
```diff
@@ -196,6 +196,9 @@
 		to_chat(SM, "<span class='warning'>All at once it makes sense: you know what you are and who you are! Self awareness is yours!</span>")
 		to_chat(SM, "<span class='userdanger'>You are grateful to be self aware and owe [user.real_name] a great debt. Serve [user.real_name], and assist [user.p_them()] in completing [user.p_their()] goals at any cost.</span>")
 		to_chat(user, "<span class='notice'>[SM] accepts [src] and suddenly becomes attentive and aware. It worked!</span>")
+		for(var/ld in GLOB.all_languages)
```
Comment (RemieRichards) [inline code/modules/research/xenobiology/xenobiology.dm]: ```SM.copy_known_languages_from(user, TRUE)```

no, I'm not kidding, this proc exists.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Yeah use the proc @RemieRichards said

**Pull request merged**